src/entity/release.rs

```diff
@@ -146,8 +147,8 @@ pub enum ReleaseStatus {
 #[derive(Debug, Serialize, Deserialize, PartialEq, Clone)]
 #[serde(rename_all(deserialize = "kebab-case"))]
 pub struct Media {
-    pub title: String,
-    pub position: u32,
+    pub title: Option<String>,
```

Comment:
@oknozor This PR introduces many `Option<T>`. I'm particularly concerned about this one.

`title` field in all other entities (say `Track`, `Recording`) aren't treated as `Option<T>` so this feels a little inconsistent and from outside it seems like we're randomly throwing `Option<T>`. "title" also sounds like an important field so I'm not sure if one would expect it to be optional (although this is for the MusicBrainz server side itself).

Maybe we could add a `Default` implementation but that would probably return an empty string for this missing `title` field as mentioned in https://github.com/oknozor/musicbrainz_rs/issues/59#issue-939558648, which I'm not sure is any better than `Option<T>`. I can't seem to think of any other way out here.

Reply:
I've been thinking about this to, it's not a matter a design on our end, the API data contains a lot of null values. I was probably wrong with replacing Option with Default, doing this would just set an alias for null value. Maybe we just need to accept using many Option...

Reply:
I see. I'll also raise this in IRC to see if anyone has any ideas. Otherwise, this is ready for merge if else everything looks good.